src/apiCalls.js

```diff
@@ -0,0 +1,21 @@
+const api = {
+
+
+  get(path) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${path}jkj`)
+      .then(response => {
+        if (!response.ok) {
+          throw `${response.status} ${response.statusText}`
```

Comment:
I like your use of `throw` here to make the code execution to run the `catch` block.